Core/include/Acts/EventData/MultiTrajectory.hpp

```diff
@@ -56,7 +56,8 @@ struct GrowableColumns {
   /// can safely be written to.
   /// @param n Number of columns to add, defaults to 1.
   /// @return View into the last allocated column
-  auto addCol(size_t n = 1) {
+  auto
+  addCol(size_t n = 1) {
```

Comment:
Yeah, this file has a ton of very short inline functions, which baloon to a pretty big file without single-line functions :/

Reply:
And this. On current 16:9 monitors, vertical space is not for free. This means there is less information on my monitor and for no good reason.